### PR TITLE
PUB-1322  - Fortnightly Press List

### PR DESCRIPTION
**Pull request by ChrisS1512**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PUB-1322

### Change description ###

Validation schema for Fortnightly Press List

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by furnivall — COMMENTED** [review]
So this may or not be relevant but Hearing Type is meant to be a mandatory field (according to Jira ticket anyway) but I can pass in a list with no hearing type sans issue. 
Sorry, I only noticed because 1320 appears to require pretty much the exact same schema!

File: src/main/resources/schemas/et_fortnightly_press_list.json [hunk 1]
```diff
@@ -0,0 +1,434 @@
...
+                                  "examples": []
+                                },
+                                "hearing": {
+                                  "$id": "#root/courtLists/courtList/courtRoom/session/sittings/items/hearing",
+                                  "type": "array",
+                                  "items": {
+                                    "type": "object",
+                                    "description": "Hearing details of the case and associated details for the sitting",
+                                    "title": "Hearing",
+                                    "required": ["case"],
```
Comment (furnivall) [inline src/main/resources/schemas/et_fortnightly_press_list.json:254]: My comment can be entirely solved by adding "hearingType" to this line.

**Review by KianKwa — APPROVED** [review]

File: src/main/resources/schemas/et_fortnightly_press_list.json (hunk 1 above)
Comment (ChrisS1512) [reply src/main/resources/schemas/et_fortnightly_press_list.json:254]: Following discussion, flagged this in the ticket. Hearing Type is marked as mandatory on the ticket, but not in the attachments.

**Review by furnivall — APPROVED** [review]

**Review by furnivall — APPROVED** [review]

**Review by furnivall — APPROVED** [review]
Wow such nice tests. I love it.,

**Review by KianKwa — APPROVED** [review]

**Pull request merged**